modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -6518,6 +6518,57 @@ type CreateGroupResult {
   group: Group!
 }
 
+  """
+  The guide star(s) and related information
+  """
+type GuideEnvironment {
+  """
+  The position angle
+  """
+  posAngle: Angle!
+
+  """
+  A list of GuideProbeTargets, which essentially provides a mapping from guide probes to targets.
+  """
+  guideTargets: [GuideTarget!]!
+}
+
+"""
+Enumeration for Guide Probes
+"""
+enum GuideProbe {
+  GMOS_OIWFS
+}
+
+"""
+Type that contains a guide probe and guide target information for use in the GuideEnvironment
+"""
+type GuideTarget {
+  """
+  The guide probe
+  """
+  probe: GuideProbe!
+
+  """
+  Target name.
+  """
+  name: NonEmptyString!
+
+  """
+  source profile
+  """
+  sourceProfile: SourceProfile!
+
+  """
+  Sidereal tracking information, if this is a sidereal target
+  """
+  sidereal: Sidereal
+
+  """
+  Nonsidereal tracking information, if this is a nonsidereal target
+  """
+  nonsidereal: Nonsidereal
```

Comment:
That was my original approach. However, a `Target` has 2 fields that didn't make sense here (IMO). 

1. The target id - since it has no independent existence and can't be edited independently, I didn't think it made sense to include the target id. 
2. The program. Since it can't be queried independent of the observation, having the program added here seemed redundant since the observation has it.

Reply:
That makes sense.  We might consider (apart from this PR) nesting a `TargetDefinition` type or something with the common information.  It just seems like the bulk of it should be shared.  If we do, that change seems like it would be beyond the scope of this PR through.